DOCS-9278-email-send-kt (#1112)
mulesoft/docs-connectors: email/1.3/modules/ROOT/pages/email-examples.adoc

> To fully benefit from using Anypoint Connector for Email (Email Connector), see the following examples to configure the connector.

Reviewer: colon before list
```suggestion
To fully benefit from using Anypoint Connector for Email (Email Connector), see the following examples to configure the connector:
```

Reviewer: So, the way "to fully benefit from using" is not really to see the examples but to ensure that you configure the connector for optimal, customized performance. Consider a wording more like this?
```suggestion
The way to fully benefit from using Anypoint Connector for Email (Email Connector) is to configure it to take advantage of and customize its functionality in your environment, as illustrated by the following examples:
```

* xref:email-send.adoc[Send Emails with Email Connector] +

> Configure Email Connector *Send* operation to send emails over SMTP or SMPTS servers.

Reviewer: You can use "the" before the name here because it goes with "operation": the Send operation.
```suggestion
Configure the Email Connector *Send* operation to send emails over SMTP or SMPTS servers.
```

Reviewer: Ditto previous formatting edits in the previous PRs, following the Style Reference entry for "font choices."

* xref:email-list.adoc[List Emails with Email Connector] +
Configure Email Connector and IMAP or POP3 protocols to list emails.
* xref:email-gmail.adoc[Connect to Gmail with Email Connector] +
Configure Email Connector and IMAP, POP3, and SMTP protocols to connect to your Gmail account.
* xref:email-trigger.adoc[Initiate a Flow When Receiving a New Email] +
Configure *On New Email - IMAP* or *On New Email - POP3* listeners to retrieve all the emails in an IMAP or POP3 mailbox, and generate a new message for each unread email found.

> 

Reviewer: Hi Kim again, I have two other PRs for Email and Database connectors examples:
Links to PRs:
https://github.com/mulesoft/docs-connectors/pull/1100/files — Database Connection example
https://github.com/mulesoft/docs-connectors/pull/1112/files — Email Send operation example
Scope: green
Type: Copyedit
Due date: Wednesday Nov 4th
Thanks!
